cmake/Modules/Platform/Emscripten.cmake

```diff
@@ -134,8 +134,6 @@ if (EMSCRIPTEN_FORCE_COMPILERS)
 	set(CMAKE_CXX_COMPILER_ID Clang)
 	set(CMAKE_CXX_STANDARD_COMPUTED_DEFAULT 98)
 
-	set(CMAKE_C_COMPILER_TARGET "asmjs-unknown-emscripten")
```

Comment:
Before we'd call `wasm-install/bin/clang -target wasm32-unknown-unknown [...] --triple asmjs-unknown-emscripten`
Now we call `wasm-install/bin/clang -target wasm32-unknown-unknown`, no `--triple`
And then watching the output for asm2wasm it calls `-target asmjs-unknown-emscripten`, and builds correctly.

There might be some case I'm not considering though.